<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="ctrl_tab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_settings">
      <attribute name="title">
       <string>Scan</string>
      </attribute>
      <layout class="QFormLayout">
       <item row="1" column="0">
        <widget class="QLabel">
         <property name="text">
          <string>K1 step size:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="ctrl_stepsize">
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
         <property name="value">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel">
         <property name="text">
          <string>Sampling points:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="ctrl_numsteps">
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QRadioButton" name="radio_active">
           <property name="text">
            <string>active value</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_zero">
           <property name="text">
            <string>zero</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel">
         <property name="text">
          <string>Scan vicinity:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel">
         <property name="text">
          <string>Number of shots:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="ctrl_numshots">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QListWidget" name="ctrl_quads">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_status">
      <attribute name="title">
       <string>Status</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QPlainTextEdit" name="ctrl_log">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="ctrl_progress">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_results">
      <attribute name="title">
       <string>Results</string>
      </attribute>
      <layout class="QHBoxLayout">
       <item>
        <widget class="TableView" name="ctrl_results">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btns">
     <property name="standardButtons">
      <set>QDialogButtonBox::Abort|QDialogButtonBox::Reset|QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>madgui/widget/tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
